static/admin/config.yml

```diff
@@ -150,25 +150,25 @@ collections:
     editor:
       preview: false
     fields:
-      - { label: "Title", name: "portfolioPageTitle", widget: "string" }
+      - { label: "Title", name: "portfolioPageTitle", widget: "string", pattern: ['.{5,60}', "Must have 5-60 characters"] }
       - label: "Projects"
         name: "projects"
         widget: "list"
         fields:
-          - { label: "Project name", name: "projectName", widget: "string" }
-          - { label: "Preview note", name: "projectPreviewNote", widget: "string" }
+          - { label: "Project name", name: "projectName", widget: "string", pattern: ['.{5,15}', "Must have 5-15 characters"] }
```

Comment:
15 characters may not be sufficient for a project name, for example, I was working on a project called NativeSpeaker.com.pl which is 20 characters :P
Maybe we can make it at least 20? Will it still look acceptable?

Reply:
I think that least 12 will be better. What do you think ?